data/json/npcs/scrap_trader/scrap_trader.json

```diff
@@ -2,14 +2,179 @@
   {
     "type": "npc",
     "id": "NPC_scrap_trader",
-    "class": "NC_SCAVENGER",
+    "class": "SCRAP_MERCHANT",
+    "gender": "male",
     "name_suffix": "scrapper",
     "attitude": 0,
-    "mission": 7,
+    "mission": 3,
     "chat": "TALK_NPC_SCRAP_TRADER",
     "faction": "wasteland_scavengers",
     "mission_offered": "MISSION_SCRAP_TRADER_GET_25_MAGNETRON"
   },
+  {
+    "type": "npc_class",
+    "id": "SCRAP_MERCHANT",
+    "name": { "str": "Merchant" },
+    "job_description": "I'm a local shopkeeper.",
+    "common": true,
+    "sells_belongings": false,
+    "bonus_str": 4,
+    "traits": [
+      { "group": "Appearance_demographics" },
+      { "trait": "STRONGBACK" },
+      { "trait": "OUTDOORSMAN" },
+      { "trait": "STRONGSTOMACH" },
+      { "trait": "PAINRESIST" },
+      { "trait": "NIGHTVISION" },
+      { "trait": "QUICK" },
+      { "trait": "GOODCARDIO" },
+      { "trait": "TOUGH" },
+      { "trait": "ADRENALINE" },
+      { "trait": "NARCOLEPTIC" },
+      { "trait": "PROJUNK" },
+      { "trait": "MOODSWINGS" }
```

Comment:
```suggestion
```
Why does the scrapper have mood swings? It seems unnecessary and problematic.

Reply:
Why is it problematic? It's just there for flavor.

Reply:
In that case, I like it.